gordo_components/cli/cli.py

```diff
@@ -387,10 +357,46 @@ def get_all_score_strings(metadata):
 
 
 @click.command("run-server")
-@click.option("--host", type=str, help="The host to run the server on.")
-@click.option("--port", type=int, help="The port to run the server on.")
-def run_server_cli(host: str, port: int):
-    server.run_server(host, port)
+@click.option(
+    "--host",
+    type=HostIP(),
+    help="The host to run the server on.",
+    default="0.0.0.0",
+    envvar="GORDO_SERVER_HOST",
+)
+@click.option(
+    "--port",
+    type=click.IntRange(1, 65535),
+    help="The port to run the server on.",
+    default=5555,
+    envvar="GORDO_SERVER_PORT",
+)
+@click.option(
+    "--workers",
+    type=PositiveInteger(),
+    help="The number of worker processes for handling requests.",
+    default=2,
+    envvar="GORDO_SERVER_WORKERS",
+)
+@click.option(
+    "--worker-connections",
+    type=PositiveInteger(),
```

Comment:
Whats the difference between these and `click.IntRange(1, some_sensible_limit)`?

Reply:
True, true... :) I'll put works at 5 and `worker_connections` at 4000, which should contain things well enough for our application here. :+1: